### ISSUE TITLE
page_compare.php should not hard-code round IDs

### ISSUE THREAD

**Issue by cpeel** [DistributedProofreaders/dproofreaders#1005]
`tools/project_manager/page_compare.php` hard-codes round values in the Comparator:
```php
        $this->L_round_options = ['P1', 'P2', 'P3', 'F1'];
        $this->R_round_options = ['F1', 'F2'];
```
As a rule we do not hard-code round IDs outside of `stages.inc` to ensure that other sites that use our code don't need to slog through the code if they have a different number of rounds (also to help future proof the code for pgdp.net if we change the number of rounds).

The Comparator should dynamically determine these in the constructor.

**Comment by 70ray** [comment]
Possibly also consider project_states.inc.

**Comment by cpeel** [comment]
> Possibly also consider project_states.inc.

Ah, yes. What's even more confusing is that `project_states.inc` needs to be included _before_ the rounds are created in `stages.inc` since the round creation coded uses the constants.

**Comment by 70ray** [comment]
> > Possibly also consider project_states.inc.
> 
> Ah, yes. What's even more confusing is that `project_states.inc` needs to be included _before_ the rounds are created in `stages.inc` since the round creation coded uses the constants.

I've made a PR to try to resolve this.

**Comment by cpeel** [comment]
Fixed in #1124
